src/schema/schema_drop.c

```diff
@@ -205,8 +205,10 @@ __drop_tiered(WT_SESSION_IMPL *session, const char *uri, bool force, const char
             session, ret = __wt_conn_dhandle_close_all(session, tier->name, true, force)));
         WT_ERR(ret);
         WT_ERR(__wt_metadata_remove(session, tier->name));
+        filename = tier->name;
+        WT_PREFIX_SKIP_REQUIRED(session, filename, "file:");
```

Comment:
You could move the computation of `filename` inside the `if (remove_files)` condition so we don't do it when it's not needed.

Reply:
Fixed.